docs/abi.md

```diff
@@ -79,7 +79,9 @@ functions** as
   reading from one of the specified channel handles. The channel handles are
   encoded in a buffer that holds N contiguous 9-byte chunks, each of which is
   made up of an 8-byte channel handle value (little-endian u64) followed by a
-  single byte that is set on return if data is available on that channel.
+  single status byte corresponding to `oak::ChannelReadStatus`. Invalid handles
+  will have an `INVALID_CHANNEL` status, but `wait_on_channels` return value
+  will only fail for internal errors.
```

Comment:
Good spot!

One pedantic point &ndash; it doesn't feel quite right to refer to a Rust SDK `enum` in an ABI document.  Could we change this to "...single channel status byte. Invalid handles..." and then also update the "Integer Types" section to describe a "**channel status**" thusly:
```md
Three specific sets of integer values are also used in the ABI:

- Many operations take a `u64` **handle** value; these values are Node-specific
  and non-zero (zero is reserved to indicate an invalid handle), and reference
  a particular [channel](concepts.md#channels).
- Many ABI operations return a `u32` **status** value, indicating the result of
  the operation. The possible values for this are defined in the `OakStatus`
  enum in [oak_abi.proto](/oak/proto/oak_abi.proto).
- The `wait_on_channels` host function fills in a **channel status** value,
  indicating the readiness status of a particular channel. The possible values
  for this are defined in the `ChannelReadStatus` enum in
  [oak_abi.proto](/oak/proto/oak_abi.proto).
```

Reply:
Done. Just checking- will the short links e.g. `/oak/proto/oak_abi.proto` work/expand correctly?

Reply:
I believe so &hellip; but I gifted you a typo: the file is oak_a**p**i.proto.